torchx/Makefile

```diff
@@ -5,11 +5,29 @@ TORCHX_SO = $(PRIV_DIR)/torchx.so
 # CMAKE_BUILD_FLAGS = --verbose
 CMAKE_BUILD_DIR ?= $(MIX_APP_PATH)/cmake
 C_SRC = $(shell pwd)/c_src
+NERVES_BUILD ?= nerves_build_no
```

Comment:
I'm not sure if we should add nerves-specific terminology to the Nx build process. @josevalim @seanmor5  thoughts?

Reply:
I'm also worried about this. But the way of building an Erlang/Elixir application leaves us (almost?) no choice...

I wonder if it's possible to move all these compatibility related code to another Elixir package...?

Reply:
I wouldn't add anything Nerves specific to this repo, but perhaps we can have an `nx_nerves` or similar which keeps these changes as a diff or something and applies them specifically when necessary